cmd/hypper/uninstall.go

```diff
@@ -15,10 +15,12 @@ var uninstallDesc = `remove a helm deployment by wrapping helm calls`
 func newUninstallCmd(actionConfig *action.Configuration, logger log.Logger) *cobra.Command {
 	client := action.NewUninstall(actionConfig)
 	cmd := &cobra.Command{
-		Use:   "uninstall [NAME]",
-		Short: "uninstall a deployment",
-		Long:  uninstallDesc,
-		Args:  require.MinimumNArgs(1),
+		Use:        "uninstall [NAME]",
+		Short:      "uninstall a deployment",
+		Long:       uninstallDesc,
+		Aliases:    []string{"del", "delete", "un"},
+		SuggestFor: []string{"remove", "rm"},
```

Comment:
oh wow, this is really nice! Very user friendly, love it!

Reply:
Straight copy-pasted from Helm! :D